src/argilla/server/policies.py

```diff
@@ -0,0 +1,69 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from argilla.server.errors import ForbiddenOperationError
+from argilla.server.models import User, UserRole, Workspace, WorkspaceUser
+
+
+class WorkspaceUserPolicy:
+    @classmethod
+    def list(cls, actor: User, workspace_user: WorkspaceUser) -> bool:
```

Comment:
Since we're using the list policy without params, it's a bit weird having the resource in the definition. Would it be easy to remove it from the method signature?

Reply:
After talking with @frascuchon we have decide to take some time to try dynamic arguments aka `kargs` for policy actions.